deploy/catalog/README.md

```diff
@@ -4,31 +4,77 @@ The Helm Chart deploys Service Catalog into an existing Kubernetes cluster.
-    to use a _local_ Docker registry.
+- [Helm](https://github.com/kubernetes/helm) must be installed in the cluster
+- Images must be built from source and pushed to a registry accessible to the
+  cluster if deploying to a non-local cluster; see the [dev guide](../../docs/DEVGUIDE.md) for instructions
 
 ## Usage
 
 Supported template parameters (values):
 
-  - `registry`  (required): Container registry with Service Catalog images.
-  - `version`   (optional): Version of Service Catalog (container images) to deploy.
-  - `namespace` (optional): A Kubernetes namespace to use for Service Catalog deployment.
-  - `storageType` (optional): The type of storage for the API server to use.
-  - `verbosity` (optional): The verbosity of logs.
-  - `debug` (optional): create a load balancer for the api server service
+### Deployment Knobs
+
+- `registry`: Container registry with Service Catalog images; optional; defaults
+  to unset
+- `namespace`: A Kubernetes namespace to use for deployment of this
+  chart into k8s; optional; defaults to `default`
+- `version`: Version of Service Catalog (container images) to deploy; optional;
+  defaults to `latest`
+- `apiServerVersion`: The version of the API server image to deploy; optional;
+  defaults to unset; overrides `version` for the API server if set
+- `controllerManagerVersion`: The version of the controller-manager image to
+  deploy; optional; defaults to unset; overrides `version` for the
+  controller-manager if set
+- `etcdImage`: The name of the etcd image to use; optional; defaults to
+  `quay.io/coreos/etcd`
+- `etcdVersion`: The version of the etcd image to deploy; optional; defaults to 
+  unset; overrides `version` for etcd if set
+
+### Catalog knobs
+
+- `verbosity`: The verbosity of logs; optional; defaults to `10`
+- `debug`: Whether to create a load balancer for the apiserver and
+  controller-manager services; optional; defaults to `false`
+- `insecure`: Whether the API server should serve insecurely; optional; defaults
+  to `false`
+- `imagePullPolicy`: The image pull policy to use for all pods in this chart;
+  optional; defaults to `Always`
+
+### API Server knobs
+
+- `insecure`: Whether the API server should serve insecurely; optional; defaults
+  to `false`
+- `insecurePort`: When `insecure` is true, the API server pod serves insecurely
+  on this container port; optional; defaults to `8081`
+- `insecureServicePort`: When `insecure` is true, the API server is fronted by a
+  k8s service serving on this node port; optional; defaults to `30001`
+- `storageType`: The type of storage for the API server to use; optional;
+  defaults to `etcd`, also accepts `tpr`
+- `globalNamespace`: The namespace to store global resources in when the API
+  server is backed by TPR
 
-Example `helm install` command:
+## Examples
+
+### Local cluster installation, backed with etcd
+
+This helm command installs into a local cluster, backing the API server with
```

Comment:
Also any reason why override the imagePullPolicy to never? Perhaps talk about it, or remove it from the command. I'm worried somebody just cuts and pastes and then new images don't load upon rebuild.